themes/cerberus/index.txt

```diff
@@ -13,8 +13,13 @@
 <% } %>
 
 <% if (locals.action) { %>
-<%- action.instructions %>
-<%- action.button.link %>
+<% if(locals.action.constructor !== Array){ %>
```

Comment:
Please move the conversion of the `locals.action` to an array to the top of the file, and condense it into one line, as follows:
```js
<% if (locals.action && locals.action.constructor !== Array) { action = [action]; } %>
```

We must do this because the plaintext version of the e-mail is sent as is, including any line-breaks added as a result of the `ejs` logic.  The less line breaks we can take up, the less line-bloated the plaintext e-mail will appear.